src/dart-2.0.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+permalink: /dart-2.0
+title: Dart 2.0 Updates
+description: How Dart 2.0 is different than 1.x, and how you can prepare.
+---
+
+This page has information about changes that are coming in Dart 2.0.
+Some changes may not apply to your target platform.
+
+* [Changes in Dart 2.0](#changes-in-dart-2.0)
+  * [Web](#web)
+  * [Mobile (Flutter)](#mobile-flutter)
+  * [Servers and command line](#servers-and-command-line)
+* [Removed from Dart 2.0](#removed-from-dart-2.0)
+
+---
+
+You can implement these changes now,
+but keep watching this page for updates. We'll let you know what you can
+do now, and what needs to wait for Dart 2.0.
+
+## Changes in Dart 2.0
+
+Dart 1.x is an optionally typed language, but Dart 2.0 requires static types.
+Thanks to [strong mode](/guides/language/sound-dart),
+the Dart analyzer can find bugs earlier in the development cycle.
+The analyzer is smart about types and can sometimes
+[infer](/guides/language/sound-dart#type-inference) a type,
+so you don't have to manually type every variable.
+
+Migrate your code to strong mode now, and you may find some bugs in the
+process! You'll also find it much easier to transition to Dart 2.0.
```

Comment:
maybe more "may identify some lurking bugs that will now be more easily identifiable and addressable" :) . -- a bit more "phew, found that sneaky bug before it hurt users" :)